libs/react/README.md

```diff
@@ -42,7 +42,7 @@ yarn
 3. Start the Storybook
 
 ```bash
-yarn storybook react
+npx run react:storybook
```

Comment:
This was the only way i managed to get storybook up and running, don't know if it is the correct way?

Reply:
It depends a little bit on what you have installed on your machine, but yes I think `npx` makes more sense to show in the docs, since it often "just works" :)

The correct command should be `npx nx run react:storybook` though!

Reply:
Changed to npx nx run react:storybook :)